src/datepicker/stateful-container.js

```diff
@@ -32,11 +33,11 @@ class StatefulContainer<T = Date> extends React.Component<
 
   constructor(props: PropsT<T>) {
     super(props);
-    const value = props.range ? [] : (null: ?T);
+    const value = props.range ? [null, null] : (null: ?T);
```

Comment:
Is it possible for this line to remain unchanged? `[]` and `[null, null]` should both resolve to the type `Array<?T>`